doc/man7/property.pod

```diff
@@ -42,9 +42,9 @@ property names like
 A I<property> is a I<name=value> pair.
 A I<property definition> is a sequence of comma separated properties.
 There can be any number of properties in a definition.
-For example: "" defines a null property definition; "my.foo=bar"
-defines a property named I<my.foo> which has a string value I<bar> and
-"iteration.count=3" defines a property named I<iteration.count> which
+For example: "" defines an empty property definition (i.e., no restriction);
```

Comment:
Did you change this??

Reply:
Yes - see commit b3e63f1eb512f34f3984769754a5cdcd8b7da40c with the message
```
Re-word null->empty property; improve iteration.count example in property.pod
```
In case the change is wrong, how best to fix it?
My understanding of the whole provider mechanism including properties etc. is still pretty rudimentary.